docs/cli.md

```diff
@@ -123,3 +123,47 @@ chronicle \
 
 Chronicle will also generate subcommands for recording provenance, derived from
 your [domain configuration](./domain_modeling.md).
+
+## Load OPA Policy from URL or File Arguments
+
+### `--opa-bundle-address --opa-policy-name --opa-policy-entrypoint`
```

Comment:
Perhaps include the arguments in angle brackets to fit with the other options this page describes.